src/types/line.js

```diff
@@ -228,7 +228,70 @@ function calculateLabelPosition(line, width, height) {
 		x = pt.x + xAdjust;
 		y = pt.y + yAdjust;
 	}
-	return {x, y};
+
+	a = {x: x - ((width / 2) * Math.cos(angle)) - ((height / 2) * Math.sin(angle)), y: y - ((width / 2) * Math.sin(angle)) + ((height / 2) * Math.cos(angle))};
```

Comment:
> Only calculating the rotated width/height here, right?

Not only, it's calculating all apexes of rotated rectangle, needed to catch also the events correctly.

And then the width and height of the rectangle are calculated using the differences among the apexes.

Reply:
@kurkle sorry, I misunderstood your comment, maybe I have understood right now. Let me test it.

Reply:
@kurkle  in `calculateLabelPosition` I need to calculate ONLY the width and height and your review is correct. I don't need to calculate the apexes because they must be calculated in `adjustLabelPosition`.

I think I need to rebase and then I'm gonna push again.